tests/pytest_tests/__init__.py

```diff
@@ -1,4 +0,0 @@
-from hypothesis import settings
```

Comment:
yuck, i didn't know we had this

Reply:
me neither... was add here: https://github.com/wandb/wandb/pull/8017
but reading the hypothesis docs https://hypothesis.readthedocs.io/en/latest/settings.html#hypothesis.settings.load_profile seems to suggest to put it in conftest so i moved it there